src/main/blockchain/staking-service.js

```diff
@@ -0,0 +1,259 @@
+import fetch from 'node-fetch';
+
+import CONFIG from 'common/config';
+import { abi as SELFKEY_ABI } from 'main/assets/data/abi.json';
+import { Token } from '../token/token';
+import BN from 'bignumber.js';
+// import { TxHistory } from './tx-history';
```

Comment:
@sk91 I think we should not leave comment code like that.

Reply:
right missed it

Reply:
@rodrigopavezi fixed